db/models.py

```diff
@@ -14,3 +14,35 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model should be something more descriptive and unique, such as `movies` or `movies_as_actor`, to avoid potential clashes with other models that might have a similar field.